File: Provas/P1_POO_2018-1/Q4/src/ContaCorrente.java
import java.util.Locale;

public class ContaCorrente {
    protected double saldo;

    public ContaCorrente () {
        this.saldo = 0;
    }

    public void depositar (double dinheiro) {
        this.saldo += dinheiro;
    }

    public void sacar (double dinheiro) {
        this.saldo -= ( dinheiro + (dinheiro*0.005) );
    }

    @Override
    public String toString () {
        return String.format(Locale.US, "Saldo = %.2f", this.saldo);
    }
}

File: Boca/CPP/05/ContaCorrente.cpp
#include "ContaCorrente.hpp"

void ContaCorrente::depositar(double quantia) {
    saldo += quantia;
}

void ContaCorrente::sacar(double quantia) {
    saldo -= quantia + (quantia * getTaxaOperacao());
}

double ContaCorrente::getSaldo() {
    return saldo;
}

double ContaCorrente::getTaxaOperacao() {
    return 0.005;
}

